src/main/java/io/thinkit/edc/client/connector/JsonLdUtil.java

```diff
@@ -26,4 +26,10 @@ public static JsonObject compact(JsonLdObject input) throws JsonLdError {
                 .build());
         return JsonLd.compact(expanded, context).get();
     }
+
+    public static JsonArray getError(InputStream body) throws JsonLdError {
```

Comment:
this could be called just `deserializeToArray` or something similar, because it's not related to the error